R/sql.R

```diff
@@ -0,0 +1,167 @@
+#' @title Run SQL queries against DataFrame/LazyFrame data.
```

Comment:
```suggestion
#' Run SQL queries against DataFrame/LazyFrame data.
```
This tag is not actually needed

Reply:
I know it is, but I used it because it appeared to be used elsewhere.